recipe/meta.yaml

```
@@ -1,33 +1,46 @@
 {% set name = "semantic_version" %}
-{% set version = "2.8.5" %}
+{% set version = "2.10.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: d2cb2de0558762934679b9a104e82eca7af448c9f4974d1f3eeccff651df8a54
+  sha256: bdabb6d336998cbb378d4b9db3a4b56a1e3235701dc05ea2690d9a997ed5041c
 
 build:
   number: 0
-  script: '{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv '
-  noarch: python
+  skip: True  # [py<35]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
 
 requirements:
   host:
     - pip
     - python
+    - setuptools
+    - wheel
   run:
     - python
 
 test:
+  requires:
+    - pip
+    - pytest
+  source_files:
+    - tests
   imports:
     - semantic_version
+  commands:
+    - pip check
+    # We skip Django-related tests because Django is not included 
+    # in the runtime dependencies. It is only used for optional development 
+    # testing (extras_require)
+    - pytest tests -v --ignore=tests/test_django.py
 
 about:
   home: https://github.com/rbarrois/python-semanticversion
-  license: BSD
+  license: BSD-3-Clause
```

```suggestion
  license: BSD-2-Clause
```
See https://github.com/rbarrois/python-semanticversion/blob/master/LICENSE